snap/local/runtime-helpers/bin/source-env-file.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -ex
```

Comment:
The script name differs from what it's called in the snapcraft file. Did you test the configuration?

Reply:
Isn't the xtrace going to pollute the logs? This script is ran several times on the snap's startup.